src/avfoundation.cs

```diff
@@ -315,10 +429,114 @@ interface AVMediaCharacteristic {
 		[Field ("AVMediaCharacteristicVoiceOverTranslation")]
 		NSString VoiceOverTranslation { get; }
 	}
+#endif
+
+	[NoWatch]
+	enum AVMetadataFormat {
+		[iOS (8,0)][Mac (10,10)]
+		[Field ("AVMetadataFormatHLSMetadata")]
+		FormatHlsMetadata = 0,
+
+		[Field ("AVMetadataFormatiTunesMetadata")]
+		FormatiTunesMetadata = 1,
+
+		[Field ("AVMetadataFormatID3Metadata")]
+		FormatID3Metadata = 2,
+
+		[Since (7,0), Mavericks]
+		[Field ("AVMetadataFormatISOUserData")]
+		FormatISOUserData = 3,
+
+		[Field ("AVMetadataFormatQuickTimeUserData")]
+		FormatQuickTimeUserData = 4,
+
+		[iOS (11,0), Mac (10,13), TV (11,0)]
+		[Field ("AVMetadataFormatUnknown")]
+		Unknown = 5,
+	}
 
+	enum AVFileTypes {
+		[Field ("AVFileTypeQuickTimeMovie")]
+		QuickTimeMovie = 0,
+		
+		[Field ("AVFileTypeMPEG4")]
+		Mpeg4 = 1,
+		
+		[Field ("AVFileTypeAppleM4V")]
+		AppleM4V = 2,
+
+		[Mac (10,11)]
+		[Field ("AVFileType3GPP")]
+	 	ThreeGpp = 3,
+		
+		[Field ("AVFileTypeAppleM4A")]
+		AppleM4a = 4, 
+		
+		[Field ("AVFileTypeCoreAudioFormat")]
+		CoreAudioFormat = 5, 
+		
+		[Field ("AVFileTypeWAVE")]
+		Wave = 6, 
+		
+		[Field ("AVFileTypeAIFF")]
+		Aiff = 7,
+		
+		[Field ("AVFileTypeAIFC")]
+		Aifc = 8, 
+		
+		[Field ("AVFileTypeAMR")]
+		Amr = 9,
+
+		[iOS (7,0), Mac (10,11)]
+		[Field ("AVFileType3GPP2")]
+		ThreeGpp2 = 10,
+
+		[Since (7,0), Mavericks]
+		[Field ("AVFileTypeMPEGLayer3")]
+		MpegLayer3 = 11,
+
+		[Since (7,0), Mavericks]
+		[Field ("AVFileTypeSunAU")]
+		SunAU = 12,
+
+		[Since (7,0), Mavericks]
+		[Field ("AVFileTypeAC3")]
+		AC3 = 13,
+
+		[iOS (9,0), Mac (10,11)]
+		[Field ("AVFileTypeEnhancedAC3")]
+		EnhancedAC3 = 14,
+
+		[iOS (11, 0), Mac (10, 13)]
+		[Field ("AVFileTypeJPEG")]
+		JPEG = 15,
```

Comment:
AFAIK I did used JPEG etc.. because we used it somewhere else, but I'll do all in CamelCase since it is a new enum.